### PR TITLE
Change Neomaril to MLOps

### PR DESCRIPTION
**Pull request by johnrc90**
## Description

Remove Neomaril reference and change it to MLOps
 
 
 
 
 
 
 
 **PR Summary by Typo**
------------

 **Summary**

This pull request includes multiple changes:

1. Renames project from "neomaril-codex" to "mlops\_codex".
2. Updates package versions and adds new SHA-256 checksums.
3. Refactors imports and class names from "Neomaril" to "MLOps".

**Key Points**

1. Renames project name and updates GitHub Actions workflow URL and settings.json file.
2. Updates package names in Pipfile and Pipfile.lock to "mlops\_codex".
3. Adds or updates SHA-256 checksums for various files and packages.
4. Refactors imports and class names from "Neomaril" to "MLOps" for better alignment with the new MLOps platform. 

 <h6>To turn off PR summary, please visit <a href="https://app.typoapp.io/settings/dev-analytics/notification?tab=codeHealth">Notification settings</a>.</h6>

### CODE REVIEW

File: notebooks/.env [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# This is a sample .env file with variables used in neomaril codex
-NEOMARIL_URL='https://neomaril.staging.datarisk.net'
-NEOMARIL_USER='email@email.com'
-NEOMARIL_PASSWORD='pass'
-NEOMARIL_GROUP_TOKEN='TOKEN'
\ No newline at end of file
+MLOPS_URL='http://localhost:7070'
+MLOPS_USER='johnrc692@gmail.com'
+MLOPS_PASSWORD='teste@123'
+MLOPS_GROUP_TOKEN='TODO'
```
Comment (henrique-lh) [inline notebooks/.env:4]: do these credentials work?

File: notebooks/Model.ipynb [hunk 2]
```diff
@@ -1,19 +1,19 @@
-{
+{MLOPSMLOPS
```
Comment (henrique-lh) [inline notebooks/Model.ipynb:1]: remove this line,  I couldn't open the notebook because of it

File: notebooks/Training.ipynb [hunk 3]
```diff
@@ -1,19 +1,19 @@
-{
+{MLOPS
```
Comment (henrique-lh) [inline notebooks/Training.ipynb:1]: same here

File: notebooks/samples/monitoring/conf.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
     "Period": "Week",
     "TrainData": {
-        "NeomarilTrainingExecution": "1",
+        "MLOpsTrainingExecution": "1",
```
Comment (henrique-lh) [inline notebooks/samples/monitoring/conf.json:4]: will it also change in the backend in the future?

Comment (johnrc90) [reply notebooks/samples/monitoring/conf.json:4]: Actually we dont send this anymore (I did a big find/replace)

File: notebooks/.env (hunk 1 above)
Comment (johnrc90) [reply notebooks/.env:4]: They work locally, but it was not supose to go there

File: notebooks/Model.ipynb (hunk 2 above)
Comment (johnrc90) [reply notebooks/Model.ipynb]: https://github.com/datarisk-io/mlops_codex/pull/64/commits/4eb4cd115b8408e34c538664bf9b9f25a99eb714

File: notebooks/.env (hunk 1 above)
Comment (johnrc90) [reply notebooks/.env]: https://github.com/datarisk-io/mlops_codex/pull/64/commits/4eb4cd115b8408e34c538664bf9b9f25a99eb714

File: notebooks/samples/monitoring/conf.json (hunk 4 above)
Comment (johnrc90) [reply notebooks/samples/monitoring/conf.json]: https://github.com/datarisk-io/mlops_codex/pull/64/commits/722fb9b1c3a4d10ced18f26d090419fb6d1cc826

Comment (henrique-lh) [reply notebooks/samples/monitoring/conf.json]: ok, is there a issue to update the samples in neomaril repo? I checked and some examples still use this variable

Comment (johnrc90) [reply notebooks/samples/monitoring/conf.json]: I think we do.

**Pull request merged**